app/controllers/api/v1/users_controller.rb

```diff
@@ -97,7 +97,7 @@ def change_password
       private
 
       def user_params
-        params.require(:user).permit(:name, :email, :password, :avatar, :language, :role_id)
+        params.require(:user).permit(:name, :email, :password, :password_confirmation, :avatar, :language, :role_id)
```

Comment:
Kind of unrelated to the PR. Couldn't create a User from admin panel without it.

Reply:
@KH-Amir-TN ?